MediaBrowser.Providers/Manager/MetadataService.cs

```diff
@@ -706,6 +706,12 @@ public bool CanRefreshPrimary(Type type)
 
                     if (localItem.HasMetadata)
                     {
+                        foreach (var remoteImage in localItem.RemoteImages)
+                        {
+                            await ProviderManager.SaveImage(item, remoteImage.url, remoteImage.type, null, cancellationToken).ConfigureAwait(false);
+                            refreshResult.UpdateType = refreshResult.UpdateType | ItemUpdateType.ImageUpdate;
```

Comment:
```suggestion
                            refreshResult.UpdateType |= ItemUpdateType.ImageUpdate;
```

Reply:
There were more of lines like this which I've changed as well. Hope it's okay.